Assets/Scripts/SpawnHandler/AnimalSpawner.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections;
+using System.Collections.Generic;
+using SpawnHandler;
+using TurnHandler.TwoPlayers;
+using UnityEngine;
+
+public class AnimalSpawner : MonoBehaviour
+{
+    [SerializeField] private List<GameObject> animalsList;
+
+    [SerializeField] private TextHandler _textHandler;
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    public void SpawnFirstTier()
+    {
```

Comment:
Move all Spawn...Tier to one method with different parameter, u can make Tier as enum